src/style.css

```diff
@@ -1,3 +1,123 @@
 body {
+  --description-and-rate-color: #060b35;
+  --code-and-price-color: #616070;
+  --cardBg-and-button-color: #fff;
+  --cardButton-color: #00acdc;
+  --common-lineHeight: 18px;
+
   margin: 0;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  background-color: #e5e5e5;
+}
+
+h2,
+p {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  height: 406px;
```

Comment:
don't hardcode the `height`
![image](https://github.com/mate-academy/layout_product-cards/assets/22824431/ec99d008-9d90-4c41-97a8-502dc8e519de)